src/test/java/seedu/address/logic/commands/CloseCommandTest.java

```diff
@@ -0,0 +1,103 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showEateryAtIndex;
+import static seedu.address.testutil.TypicalEateries.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalFeeds.getTypicalFeedList;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_EATERY;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_EATERY;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.eatery.Eatery;
+import seedu.address.testutil.EateryBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for
+ * {@code CloseCommand}.
+ */
+public class CloseCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), getTypicalFeedList(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Eatery closedEatery = new EateryBuilder().withTags("friends").buildClose();
```

Comment:
same comment as the in your eaterybuilder, why buildClose() instead of .withIsOpen(false).build()?